src/main/java/store/bookshop/repository/BookRepository.java

```diff
@@ -1,8 +1,10 @@
 package store.bookshop.repository;
 
+import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
 import store.bookshop.model.Book;
 
 public interface BookRepository extends JpaRepository<Book, Long> {
 
+    List<Book> findByCategories_Id(Long id);
```

Comment:
```suggestion
    List<Book> findByCategoryId(Long categoryId);
```

Reply:
Reason: Failed to create query for method public abstract java.util.List store.bookshop.repository.BookRepository.findByCategoryId(java.lang.Long); No property 'categoryId' found for type 'Book'
 
 I need a method with exactly the same name so that it can parse it correctly and create the correct query automatically, or I can change the name and manually write a query